core/types/block.go

```diff
@@ -169,7 +179,7 @@ type Body struct {
 type Block struct {
 	header       *Header
 	uncles       []*Header
-	transactions Transactions
+	transactions []*Transaction
```

Comment:
Oops, I changed the `extblock` to use `Transactions` instead of `[]*Transaction` in an earlier version to hijack `EncodeRLP`, but when I manually reverted I accidentally also changed this one. I will undo this.